spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -183,6 +183,12 @@ const ConfigSteps = () => {
 		navigate("/spa/connected");
 	};
 
+	const installNewOrg = async () => {
+		await OAuthManagerInstance.installNewApp(() => {
```

Comment:
/nit
```suggestion
		await OAuthManagerInstance.installNewApp(getOrganizations)
```

Reply:
Actually I leave this as it was, feels it could be more flexible in future.